client/src/globalState.tsx

```diff
@@ -328,7 +328,7 @@ export const GlobalStateProvider: React.FC<GlobalStateProviderProps> = ({
     userEmail,
     userLookup,
 
-    activeTab,
+    activeTab: payloadToBeSent ? ChatTab : activeTab,
```

Comment:
I don't like this, this stops you switching tabs entirely when you have a payload ready to send without any ui feedback. I think it'd be better to set the chat as the active tab when a payload is added but after that allow the user to switch back and forth as they want.